spec/models/export/activity_implementing_organisation_column_spec.rb

```diff
@@ -0,0 +1,105 @@
+RSpec.describe Export::ActivityImplementingOrganisationColumn do
+  before(:all) do
+    DatabaseCleaner.strategy = :transaction
+    DatabaseCleaner.start
+
+    @activity_with_single_organisation = activity_with_single_organisation
+    @activity_with_multiple_organisations = activity_with_multiple_organisations
+    @level_b_activity_with_extending_organisation = level_b_activity_with_extending_organisation
+  end
+
+  after(:all) do
+    DatabaseCleaner.clean
+  end
+
+  context "when there are activities" do
+    subject { described_class.new(activities_relation: Activity.all) }
+
+    describe "#headers" do
+      it "returns the correct header" do
+        expect(subject.headers).to match_array(["Implementing organisations"])
+      end
+    end
+
+    describe "#rows" do
+      context "when the acitivity is level C (project)" do
```

Comment:
tiny typo in "acitivity"

Reply:
You would think I could type 'activity' by now! 🤦